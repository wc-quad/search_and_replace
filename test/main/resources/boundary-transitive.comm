R flee flea
R flea fly
R fly flue
Q
