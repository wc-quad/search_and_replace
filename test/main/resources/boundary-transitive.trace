Give a sequence of commands of the form:
s word             [search]
r word replacement [replace]
Type 'q' to quit
< Said the fly, "let us flee!"
> Said the fly, "let us flea!"
< A flea and a fly in a flue
> A fly and a fly in a flue
< Said the fly, "let us flea!"
> Said the fly, "let us fly!"
< "Let us fly!" said the flea.
> "Let us fly!" said the fly.
< A fly and a fly in a flue
> A flue and a flue in a flue
< Said the fly, "let us fly!"
> Said the flue, "let us flue!"
< "Let us fly!" said the fly.
> "Let us flue!" said the flue.
