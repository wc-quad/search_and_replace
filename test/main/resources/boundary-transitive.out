A flue and a flue in a flue
Were imprisoned, so what could they do?
Said the flue, "let us flue!"
"Let us flue!" said the flue.
So they flew through a flaw in the flue.
